### DISCUSSION TITLE
Array tag on table element

### DISCUSSION THREAD

**Discussion by Enromi** [frangoteam/FUXA#1336] [Q&A]
Hi!
I'm pretty new to this and I'm fighting with the table element.
I am reading a series of variables from an OPC UA server and one of them is an array with variable rows, where each element contains 4 fields (struct array). I would like it to be represented in a table, with its fields as columns and the number of rows changing depending on the ones contained in the array. Would it be possible to do that?

Thank you very much!

**Comment by unocelli** [comment]
Hi, I think that with the script it can already be done
